rotkehlchen/tests/db/test_history_events.py

```diff
@@ -94,6 +94,7 @@ def add_history_events_to_db(db: DBHistoryEvents, history_data: dict) -> None:
                     asset=A_ETH,
                     balance=Balance(entry[2]),
                 ),
+                mapping_values=entry[3],
```

Comment:
can you add a small docstring explaining what this funciton does? and also maybe instead of using `dict` in the type we could define `dict[str, tuple[add here the types]` and improve it

Reply:
Updated as suggested.